### DISCUSSION TITLE
JScontents/obj_frozen/

### DISCUSSION THREAD

**Discussion by giscus** [yeopto/yeopto.github.io#11] [Comments]
# JScontents/obj_frozen/

중첩 객체까지 동결할 수 있는 방법

https://yeopto.github.io/JScontents/obj_frozen/

**Comment by minuet1215** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ
두 번이나 언급해주심에 무한한 감사를

**Comment by minuet1215** [comment]
댓글창 복귀 축하드립니다.😃
